proto-lens-tests/package.yaml

```diff
@@ -209,6 +209,17 @@ tests:
       - Proto.Any
       - Proto.Any_Fields
 
+  decode_delimited:
```

Comment:
Nit: please follow the convention of the other tests and name this "decode_delimited", and name the corresponding source file "decode_delimited_test.hs".

Reply:
Sorry, I didn't get: is the name of the test wrong too? The name of the _file_ I will fix, thanks!

Reply:
Yeah, by convention the test names in package.yaml also have "_test" at the end.